src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+public class Ui {
+    private final static String INDENT_LINE = "____________________________________________________________";
+
+    /**
+     * Prints a given String in a format characteristic of the Ui
+     * @param toDisplay the String to be displayed in the formatted manner
+     */
+    private void printFormatted(String toDisplay) {
+        System.out.println(INDENT_LINE + "\n" + toDisplay + "\n" + INDENT_LINE);
+    }
+
+    /**
+     * Returns the String representation of the list of tasks
+     * @param taskList the list of tasks
+     * @return the String representation of the list of tasks
+     */
+    private String getTaskList(TaskList taskList) {
+        String taskListString = "Here are the tasks in your list:\n";
+        for (int i = 0; i < taskList.size() - 1; i++) {
+            taskListString += (i + 1) + "." + taskList.get(i) + "\n";
+        }
+        taskListString += (taskList.size() + "." + taskList.get(taskList.size() - 1));
+        return taskListString;
+    }
+
+    /**
+     * Prints the list of tasks
+     * @param taskList the list of tasks
+     */
+    public void showTaskList(TaskList taskList) {
+        printFormatted(getTaskList(taskList));
+    }
+
+    /**
+     * Prints the initial message characteristic of the Ui
+     */
+    public void showInitMessage() {
+        printFormatted("Hello! I'm Vincent\n" + "What can I do for you?");
+    }
+
+    /**
+     * Prints a message on encountering the "bye" command
+     */
+    public void showByeMessage() {
+        printFormatted("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Prints a message on Task deletion
+     * @param deletedTask the deleted Task
+     * @param len the length of the list of tasks after deletion
+     */
+    public void showDeleteMessage(Task deletedTask, int len) {
+        printFormatted("Noted. I've removed this task:\n" + deletedTask + "\nNow you have " +
+                len + " tasks in the list.");
```

Comment:
should the line break happen before the plus sign? i.e.
printFormatted("Noted. I've removed this task:\n" + deletedTask + "\nNow you have "
                + len + " tasks in the list.");